src/config.ts

```diff
@@ -37,6 +37,13 @@ export interface SwaggerConfig {
    */
   outputDirectory: string;
 
+  /**
+   * Base-name of swagger.json or swagger.yaml.
+   *
+   * Default: "swagger"
+   */
+  outputBasename?: string;
```

Comment:
For routes, this is called `routesFileName`, so we should call this:
```suggestion
  specFileName?: string;
```

Reply:
`routesFileName` is a full filename -- it defaults to `routes.ts`.

This variable has `.json` or `.yaml` appended, which is why I didn't suffix with `FileName`.

(Still want this change?)

Reply:
Updated my suggestion, if it's not too much to ask change it, otherwise I'm fine with outputBasename aswell